packages/client/test/client.spec.ts

```diff
@@ -80,7 +78,7 @@ describe("Client Integration", () => {
       const clients = await initTwoClients();
       const fakeTopic = "nonsense";
       await expect(clients.A.ping({ topic: fakeTopic })).to.eventually.be.rejectedWith(
-        `No matching pairing or session with topic: ${fakeTopic}`,
+        "Missing or invalid topi",
```

Comment:
Typo?

Reply:
Yep, fixed 🆗 at some point will prob use `ERROR.SOMETHING.stringify()` for tests as well so its all consistent